src/v/cluster/controller.cc

```diff
@@ -538,8 +546,27 @@ ss::future<> controller::create_cluster() {
         }
 
         co_await ss::sleep_abortable(retry_jitter.next_duration(), _as.local());
-        vlog(clusterlog.trace, "Retrying to create cluster {}", cmd_data.uuid);
+        vlog(
+          clusterlog.trace, "Will retry to create cluster {}", cmd_data.uuid);
+    }
+}
+
+ss::future<ss::stop_iteration> controller::seeds_driven_leader_barrier() const {
```

Comment:
Could we be more descriptive with the name here? Maybe something like `wait_for_leadership_or_cluster_uuid()` or something? Or see my other idea of just inlining this into `create_cluster()`

Reply:
function removed